### PR TITLE
docs: add `Skeleton` documentation

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: apps/docs/src/pages/components/skeleton.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+### Custom animation
+
+You can use custom CSS animations with the skeleton component by setting the `animation` prop to `undefined`
+and defining your own in your styles.
+
+```tsx live
+export default function Demo() {
+    import { useMemo } from "react";
+
+    const customAnimationKeyframes = useMemo(
```
Comment (zettca) [inline apps/docs/src/pages/components/skeleton.mdx:85]: these should be outside the component

File: apps/docs/src/pages/components/skeleton.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      () => `
+      @keyframes customAnimation {
+        50% { box-shadow: inset 0px 0px 60px 0px rgba(0,0,0,0.25); }
```
Comment (zettca) [inline apps/docs/src/pages/components/skeleton.mdx:88]: this is not visible in dark theme; can we use a color that works in both?
```suggestion
        50% { box-shadow: inset 0px 0px 60px 0px #66669940; }
```

File: apps/docs/src/pages/components/skeleton.mdx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      <style>{customAnimationKeyframes}</style>
+      <HvSkeleton
+        width={300}
+        height={100}
+        animation={undefined}
+        variant="square"
+        style={{
+          animation: "customAnimation 1s infinite"
+        }}
+      />
```
Comment (zettca) [inline apps/docs/src/pages/components/skeleton.mdx:106]: Why not use `css()` for the custom animation, like we're doing elsewhere?
If we're avoiding Emotion, then we could leverage UnoCSS's built-in animations (eg. `animate-flash`); what do you think?

Also, `animation={undefined}` does nothing

```suggestion
      <HvSkeleton
        width={300}
        height={100}
        variant="square"
        className="animate-flash animate-count-infinite"
      />
```

**Comment by plagoa** [comment]
@zettca updated PR.

File: apps/docs/src/pages/components/skeleton.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+and defining your own in your styles.
+
+```tsx live
+<HvSkeleton
+  width={300}
+  height={100}
+  animation={undefined}
+  variant="square"
+  className="animate-flash animate-count-infinite"
+/>
```
Comment (zettca) [inline apps/docs/src/pages/components/skeleton.mdx:88]: We'll need the `!` or the animation won't work.
We should probably review the `HvProvider`'s and UnoCSS's injection order

```suggestion
You can use custom CSS animations with the skeleton component by defining your own in your styles.

```tsx live
<HvSkeleton
  width={300}
  height={100}
  variant="square"
  className="!animate-flash !animate-count-infinite"
/>
```

Comment (plagoa) [reply apps/docs/src/pages/components/skeleton.mdx]: Updated.

Comment (zettca) [reply apps/docs/src/pages/components/skeleton.mdx]: `animation={undefined}` and "by setting the `animation` prop to `undefined`" is still there

Comment (plagoa) [reply apps/docs/src/pages/components/skeleton.mdx]: Updated.

**Pull request merged**